src/site/content/en/metrics/inp/index.md

```diff
@@ -116,6 +116,10 @@ As far as INP goes, **only the following interaction types are observed:**
 - Tapping on a device with a touchscreen.
 - Pressing a key on either a physical or onscreen keyboard.
 
+{% Aside 'important' %}
+Hovering and scrolling does not factor into INP. However, scrolling with the space bar counts as a keystroke and _is_ factored into INP, as the main thread may block key inputs in the presence of long tasks.
```

Comment:
Continuous events like hovering and scrolling are not measured by INP.

However, sometimes keyboard everts (space bar, arrow buttons, PgUp PgDown) can start scrolling.  The scroll itself is not measured, but INP will measure the keyboard events themselves.

That is because long tasks on main thread may block the keyboard event which needs to be processed before you can even start the scroll.  The scroll itself is never measured.

Note, JavaScript like the following can prevent scrolling via keyboard, so its clear that the INP is just for the key itself not the scroll

```
document.addEventListener('keydown', (evt) => {
    evt.preventDefault();
});
```

Reply:
urgh—I already merged this one, but I can open a second PR to address these. Sorry I got all merge-happy!